Solutions/BrianCarName.java

```diff
@@ -0,0 +1,48 @@
+import java.util.HashMap;
+
+/**
+ * This class is responsible to check whether Brian Car name is OK or not.
+ * Car name is OK only if it satisfies few conditions.
+ * 
+ * @author kumar Kaushal
+ * 
+ */
+public class BrianCarName {
+
+	/**
+	 * This method checks whether Brian's car name is OK or not. It returns OK
+	 * only if satisfies the following conditions Name must Consist exactly
+	 * three distinct characters All distinct characters must occur same number
+	 * of times i.e. if C1 occurred 2 times, C2 and C3 must also occur 2 times.
+	 * 
+	 * @param str
+	 * @return true or false
+	 */
+	public String checkBrianCarName(String inputString) {
+		String isBrianCarNameOk = "OK";
```

Comment:
What other name can we give for this variable? isBrianCarNameOk

Reply:
isBrianCarNameOk is not a proper name for String type should be changed carNameStatus